packages/ui-utils/src/types.ts

```diff
@@ -463,3 +464,8 @@ export type DataMessage = {
   role: 'data';
   data: JSONValue; // application-specific data
 };
+
+export type AssistantStreamPart = {
+  event: AssistantThreadStatus;
+  data?: any;
+};
```

Comment:
most stream parts are defined in stream parts afaik. do we need to expose this? otherwise would prefer if it's consistent with the other stream parts. also, `any` is risky. is `unknown` possible?

Reply:
updated to `unknown` and also defined it inside `stream-parts`!